pkg/operator/ceph/object/admin.go

```diff
@@ -18,13 +18,19 @@ package object
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/pkg/errors"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/clusterd"
 	"github.com/rook/rook/pkg/daemon/ceph/client"
 )
 
+const (
+	// rgwCmdTimeout is the timeout used when running rgw CLI commands
+	rgwCmdTimeout = 1 * time.Minute
```

Comment:
What about a timeout of 30s? That's still more than enough time for normal operations.

Reply:
Actually, what about sticking with the 15s default we have for all the `ceph` commands?

https://github.com/rook/rook/blob/31bcb60302dbb15de7f7e71b4f0a6b68f209c7a9/pkg/daemon/ceph/client/command.go#L44

Reply:
using 15 seconds.